test_pydantic_schemas.py

```
@@ -0,0 +1,409 @@
+        print("✅ Serialization/deserialization working correctly")
+        print("✅ Response models working correctly")
+        print("✅ Error handling working correctly")
+    else:
+        print("❌ Some schema tests failed")
+    
+    return success
+
+
+if __name__ == "__main__":
+    success = main()
+    sys.exit(0 if success else 1)
```

Ruff (0.12.2)

No newline at end of file

Add trailing newline

(W292)